tests/unit/triagers/plugins/test_shipit.py

```diff
@@ -975,3 +1069,74 @@ def test_automerge_changelog_fragment(self):
         afacts = get_automerge_facts(iw, meta)
 
         self.assertTrue(afacts[u'automerge'])
+
+    def test_automerge_deletion_from_ignore(self):
+        iw = IssueWrapperMock('ansible', 'ansible', 1)
+        iw._is_pullrequest = True
+        mfile = MockFile(u'test/sanity/validate-modules/ignore.txt')
+        mfile.deletions = 1
+        iw.pr_files = [mfile]
+        meta = {
+            u'is_module_util': False,
+            u'is_new_module': False,
+            u'is_needs_rebase': False,
+            u'is_needs_revision': False,
+            u'component_support': [u'community'],
+            u'is_backport': False,
+            u'merge_commits': False,
+            u'has_commit_mention': False,
+            u'is_needs_info': False,
+            u'has_shippable': True,
+            u'has_travis': False,
+            u'mergeable': True,
+            u'ci_stale': False,
+            u'ci_state': u'success',
+            u'shipit': True,
+            u'supershipit': False,
+            u'is_new_directory': False,
+            u'is_module': True,
+            u'module_match': {
+                u'namespace': u'foo',
+                u'maintainers': [u'ghuser1'],
+            },
+        }
+
+        afacts = get_automerge_facts(iw, meta)
+
+        self.assertTrue(afacts[u'automerge'])
+
+    def test_automerge_addition_to_ignore(self):
+        iw = IssueWrapperMock('ansible', 'ansible', 1)
+        iw._is_pullrequest = True
+        mfile = MockFile(u'test/sanity/validate-modules/ignore.txt')
+        mfile.addtions = 1
```

Comment:
Typo?

```suggestion
        mfile.additions = 1
```

Also, if this test is passing now, that would seem to indicate we're not testing what we think we are.